ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -242,18 +213,17 @@ open class TableViewCell: UITableViewCell {
     @objc public static let defaultNumberOfLinesForLargerDynamicType: Int = -1
 
     /// The default horizontal spacing in the cell.
-    @objc public static let horizontalSpacing: CGFloat = 16
+    @objc public var horizontalSpacing: CGFloat { tokens.horizontalSpacing }
 
     /// The default leading padding in the cell.
-    @objc public static let defaultPaddingLeading: CGFloat = horizontalSpacing
+    @objc public var defaultPaddingLeading: CGFloat { tokens.paddingLeading }
 
     /// The default trailing padding in the cell.
-    @objc public static let defaultPaddingTrailing: CGFloat = horizontalSpacing
+    @objc public var defaultPaddingTrailing: CGFloat { tokens.paddingTrailing }
```

Comment:
Changing this from static to an instance property will break the consumers anyways.
What is the rationale behind it? Is it about retrieving the default values of the tokens of the window associated theme?

Reply:
The `static` vars were useful for consumers who wanted to build their own custom UIs that looked like a Fluent `TableViewCell` but was not in fact one. Now that these values are token-backed, I see less value in having them available as public APIs at all, when callers could just as easily query these properties from a token instance.

Thoughts?

Reply:
I agree that these values could be queried directly from the tokens. But in this case they're not necessarily the "default" options, right? It'll return whatever the tokens are for that specific TableViewCell.

Are these vars still here just for the ObjC availability?

Reply:
Yes they are only available for static use in ObjC, but unfortunately that makes it difficult to work with tokens

Reply:
Resolved offline: users should no longer need the static variables as they are accessible from the tokens.